task/testmap.py

```diff
@@ -141,8 +141,6 @@
             'fedora-31/firefox',
             'rhel-7-8/firefox',
             'rhel-8-1/chrome',
-        ],
-        'rhel-8.2': [
```

Comment:
Please just remove the section. rhel-8-2 is already being tested on master. We don't need to test all browsers there, pairwise testing across browsers and OSes is enough IMHO.

Reply:
Of course. Updated.